frontend/src/metabase/querying/components/TimeseriesChrome/TimeseriesChrome.unit.spec.tsx

```diff
@@ -0,0 +1,100 @@
+import { userEvent } from "@storybook/testing-library";
+
+import { renderWithProviders, screen } from "__support__/ui";
+import * as Lib from "metabase-lib";
+import {
+  createQuery,
+  createQueryWithClauses,
+  SAMPLE_METADATA,
+} from "metabase-lib/test-helpers";
+import Question from "metabase-lib/v1/Question";
+import { createMockCard } from "metabase-types/api/mocks";
+
+import { TimeseriesChrome } from "./TimeseriesChrome";
+
+interface SetupOpts {
+  query?: Lib.Query;
+}
+
+function setup({ query = createQuery() }: SetupOpts = {}) {
+  const question = new Question(createMockCard(), SAMPLE_METADATA).setQuery(
+    query,
+  );
+  const stageIndex = -1;
+  const updateQuestion = jest.fn();
+
+  const getNextBreakouts = () => {
+    const nextQuestion = updateQuestion.mock.calls[0][0];
+    const nextQuery = nextQuestion.query();
+    return Lib.breakouts(nextQuery, stageIndex).map(breakout =>
+      Lib.displayInfo(nextQuery, stageIndex, breakout),
+    );
+  };
+
+  renderWithProviders(
+    <TimeseriesChrome question={question} updateQuestion={updateQuestion} />,
+  );
+
+  return { getNextBreakouts };
+}
+
+describe("TimeseriesChrome", () => {
+  it("should render nothing if there are no breakouts", () => {
+    setup();
+    expect(screen.queryByText("View")).not.toBeInTheDocument();
+  });
+
+  it("should render nothing if there are no breakouts on a temporal column", () => {
+    const query = createQueryWithClauses({
+      breakouts: [{ tableName: "PRODUCTS", columnName: "CATEGORY" }],
+    });
+    setup({ query });
+    expect(screen.queryByText("View")).not.toBeInTheDocument();
+  });
+
+  it("should allow to change the temporal unit for a breakout", async () => {
+    const query = createQueryWithClauses({
+      breakouts: [
+        {
+          tableName: "PRODUCTS",
+          columnName: "CREATED_AT",
+          temporalBucketName: "Month",
+        },
+      ],
+    });
+    const { getNextBreakouts } = setup({ query });
+
+    await userEvent.click(screen.getByText("Month"));
+    await userEvent.click(await screen.findByText("Year"));
+
+    expect(getNextBreakouts()).toMatchObject([
+      { displayName: "Created At: Year" },
+    ]);
+  });
+
+  it("should allow to change the temporal unit for a breakout when there are multiple breakouts of the same column", async () => {
```

Comment:
1. Remove the `breakoutPositions.length === 0` as mentioned in PR description
2. New > Question > Orders
3. Add a breakout "Created At: Month"
4. Add another breakout "Created At: Year"
5. Visualize
6. Change bucket to "Year" using timeseries chrome

Actual: all breakouts disappear
Expected: noop? UI should not allow user to do that?

Reply:
:heavy_check_mark:  Looks like it's [been fixed](https://github.com/metabase/metabase/pull/46555#discussion_r1711903238).